validator/client/src/main/java/tech/pegasys/teku/validator/client/ProposerConfig.java

```diff
@@ -121,6 +121,10 @@ public Optional<Eth1Address> getFeeRecipient() {
       return Optional.ofNullable(feeRecipient);
     }
 
+    public Optional<UInt64> getGasLimit() {
+      return Optional.ofNullable(builder.gasLimit);
+    }
+
```

Comment:
my guess is we could clean this up by storing optionals, but I haven't looked at all the potential knock-on effects here

Reply:
Actually there is an `Optional<UInt64> getGasLimit()` already. I reused it

Reply:
Changing the builder config with optional fields would a big separate PR. For now I just reused the getter that wraps up the value into an optional...